src/Makefile.am

```diff
@@ -123,6 +123,7 @@ libdmtcpinternal_a_SOURCES = coordinatorapi.cpp 		\
 			     procselfmaps.cpp			\
 			     rwlock.cpp 			\
 			     shareddata.cpp 			\
+				 threadsync.cpp             \
```

Comment:
FORMATTING:
The spacing here for threadsync.cpp seems to be off.  There should be 3 tabs instead of 4, and the final backslash is missing the two tabs b efore it.